.circleci/config.base.yml

```diff
@@ -749,7 +780,7 @@ jobs:
             version=$(cat .amplify-pkg-version)
             yarn ts-node scripts/github-release.ts $version $commit
   cleanup_resources:
-    <<: *linux-e2e-executor
+    <<: *linux-e2e-executor-large
```

Comment:
clenup_resources and cleanup_resources_after_e2e_runs are likely IO bound. could you try scaling down?

Reply:
similar tradeoff here; we can downscale but the savings are minimal; prefer to speed this job up during spinup/teardown

Reply:
for reference, 1 hour of large executor cost about $.72 vs $.36 for small executor

Reply:
I wouldn't trade time for pennies either.
The suggestion is to try running these jobs (and from other comment) on smaller size and see if size makes any difference for them. If there isn't then downsize. if there is then keep it large.